elfo-core/src/memory_tracker.rs

```diff
@@ -41,21 +58,24 @@ mod proc_stats {
 
     use super::MemoryStats;
 
+    const PROC_MEMINFO: &str = "/proc/meminfo";
+
     pub(super) fn get() -> Result<MemoryStats, String> {
         const PROC_SELF_STATM: &str = "/proc/self/statm";
-        const PROC_MEMINFO: &str = "/proc/meminfo";
         const PAGE_SIZE: usize = 4096; // TODO: use `sysconf(_SC_PAGESIZE)`
 
-        // Get `total`.
+        // Assumes `MemTotal` is always the 1st line in `/proc/meminfo`.
+        const MEM_TOTAL_LINE_NO: usize = 0;
+        // Assumes `MemAvailable` is always the 3rd line in `/proc/meminfo`.
+        const MEM_AVAILABLE_LINE_NO: usize = 2;
```

Comment:
I don't think we should rely on the exact position in the file.

Reply:
Noted, changed